command/watch.go

```diff
@@ -182,11 +185,19 @@ func (c *WatchCommand) Run(args []string) int {
 			cmd.Stdout = os.Stdout
 			cmd.Stderr = os.Stderr
 
+			// Watch for a stop
+			go func() {
+				<-wpStoppingCh
```

Comment:
I think you could also just read off of the `c.ShutdownCh` here, that way the extra channel is not needed.